elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -111,25 +111,14 @@ class PfeContentSet extends PFElement {
     return this.parentNode ? this.parentNode.offsetWidth > breakpointValue : window.outerWidth > breakpointValue;
   }
 
-  get tab() {
-    // Check if the appropriate tag exists already
+  get tabs() {
```

Comment:
Updated this naming since it's capturing the tabs wrapper instead of an individual tab